draft-ietf-teep-protocol.md

```diff
@@ -527,7 +527,7 @@ update = [
   type: TEEP-TYPE-update,
   options: {
     ? token => uint,
-    ? tc-list => [ + bstr ],
+    ? tc-list => [ + tc-info ],
```

Comment:
This was incorrect before (thanks for catching), but tc-info is not the correct type.  See my next comment.

Reply:
The issue for tracking the discussion is https://github.com/ietf-teep/teep-protocol/issues/95.
Closing this PR, the new PR is #97